commands/_internals/loadProjectDotdde.sh

```diff
@@ -0,0 +1,32 @@
+function _loadProjectDotdde {
+
+    # Define the filename for the .dde.yml file
+    DDE_FILE=".dde.yml"
+
+    # Check if the .dde.yml file exists in the current directory
+    if [ ! -f "$DDE_FILE" ]; then
+        _logYellow "File $DDE_FILE not found. Continuing without loading .dde.yml."
+    else
+        # Read the version number from the .dde.yml file using the _yq_stdin function
+        DDE_VERSION=$(_yq_stdin e '.version' < "$DDE_FILE")
+
+        case $DDE_VERSION in
+            1)
```

Comment:
@sbaerlocher Can we version this more fluently? It should be compatible with semver and allow us to only override whats changed, without redefining everything